### DISCUSSION TITLE
So we still have to build by self as of today?

### DISCUSSION THREAD

**Discussion by deostroll** [tenacityteam/tenacity-legacy#607] [Q&A] [answered]
Normally I am ok about this. But right now I feel as if my machine would freeze if I go about building tenacity on windows. So is there a place where I can find already find the built software for windows at least. Further more are the build machine specs the same as the software system requirements specs?

So now what is different? My exe will still bear the name audacity, is it? Any other similar differences I should be aware of?

**Comment by unbeatable-101** [comment, marked as answer]
https://tenacityaudio.org/very-early-builds

**Reply by deostroll** [reply]
These builds don't have the main executable file in them.

**Reply by Semisol** [reply]
> These builds don't have the main executable file in them.

What? The zip when unpacked gives an .exe
